web/client/components/geostory/media/Image.jsx

```diff
@@ -109,12 +108,6 @@ export default compose(
                     return resource.data;
                 }
             )
-        ),
-        emptyState(
```

Comment:
Removing this causes issues when add a media and then close the media editor. If you wanted to avoid this to appear in empty background, we should create a default empty content specific for backgrounds.

Reply:
sorry, which issue is causing exaclty?

Reply:
should be fixed now